tron/mesos.py

```diff
@@ -328,13 +334,18 @@ def submit(self, task):
 
         mesos_task_id = task.get_mesos_id()
         self.tasks[mesos_task_id] = task
-        self.runner.run(task.get_config())
-        log.info(
-            'Submitting task {} to {}'.format(
-                mesos_task_id,
-                self.mesos_address,
-            ),
-        )
+        if recover:
```

Comment:
what do you think about moving this into a separate ``recover`` function on the task, rather than overloading this function?

Reply:
Also done.